homeassistant/components/device_automation/toggle_entity.py

```diff
@@ -56,11 +58,13 @@
         # Trigger when entity is turned off
         CONF_PLATFORM: "device",
         CONF_TYPE: CONF_TURNED_OFF,
+        CONF_SUPPORTS: [CONF_FOR],
```

Comment:
I think that this is kinda weird. Now we're conflating configuration of a trigger and capabilities of a trigger. Would it make more sense to add `CONF_FOR: None`, which indicates in the frontend that it supports `FOR` but that none is set ?

Reply:
I agree this is weird, it means a trigger created with an old version of the backend will not gain new capabilities if its edited.

I did try `CONF_FOR: None` and it is also not so nice:
- We can't simply use `onChangeEvent` in frontend to set value based on an input as it will remove the key if it's empty
- Backend needs to take special care to ignore `None`, `""` etc. from the frontend
- We're mixing configuration and capability of the trigger in the same key

Other options:
- The frontend asks the backend about capability of a trigger, based on its `device_id`, `domain` and `type`.
- The list of triggers from the frontend is changed from
`[trigger_0, trigger_1, ... trigger_n]` 
to:
`[trigger_0: {trigger, capabilities}, trigger_1: {trigger, capabilities}, ... trigger_n: {trigger, capabilities}]`

Reply:
Actually, we wouldn't need capaibilities _that_ often, so we could make it a separate API.

Reply:
Refactored to add new WS API `device_automation/trigger/capabilities`